docs/user-guide/bulk-update/bulk-update.md

```diff
@@ -0,0 +1,72 @@
+# Bulk Update
+
+## OVERVIEW
+
+This feature helps you to update deployment template for multiple apps in one go! You can filter the apps on the basis
+of environments, global flag, and app names(we provide support for both substrings included and excluded in the app
+name).
+
+## Script Execution
+
+Click on Bulk edit button at the left panel. To run a script, add the details as mentioned in sample script.
+
+![](../../.gitbook/assets/bulk-update-1.png)
+
+For Run script result go to the bottom section, there is a tab `Output` you can check bulk update response here.
```

Comment:
To execute the script click on `Run` button on the top. Result can be viewed in the `Output` tab at the bottom.